### PR TITLE
Make EuiTableSortMobile render EuiTableSortMobileItems

### PR DESCRIPTION
**Pull request by jen-huang**
Make EuiTableSortMobile render EuiTableSortMobileItems, so that popover can be closed. Popover not closing was making the custom table mobile sorting appear laggy.

Make loading & loading error message centered like on desktop view.

### CODE REVIEW

File: src/components/table/mobile/table_sort_mobile.js [hunk 1]
```diff
@@ -65,14 +63,29 @@ export class EuiTableSortMobile extends Component {
         ownFocus
         button={mobileSortButton}
         isOpen={this.state.isPopoverOpen}
-        closePopover={this.closePopover.bind(this)}
-        anchorPosition={anchorPosition || "downRight"}
```
Comment (cchaos) [inline src/components/table/mobile/table_sort_mobile.js]: Any particular reason not to allow this prop to be passed optionally?

**Review by cchaos — COMMENTED** [review]
Thank you so much for this! The sorting is definitely much snappier. Quick question, is it possible not to have the popover auto-close on selection? It's nice to give the user the feedback of knowing what they clicked on without having to back into the popover. My other thought would be to change that button text to "Sort by: {columnHeader}". Is it possible to pass that in?

File: src/components/table/mobile/table_sort_mobile.js (hunk 1 above)
Comment (jen-huang) [reply src/components/table/mobile/table_sort_mobile.js]: I had a good reason at some point 😆will add back

**Comment by jen-huang** [comment]
I was running into an issue where the contents of the popover wouldn't change until it was closed and re-opened. I'll take a closer look.

**Comment by jen-huang** [comment]
Hmm. I rebased my branch with upstream master. This appears to be no bueno since your branch isn't updated yet. Closing this PR, will make a new one shortly.

**Pull request closed without merging**